content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,317 @@
+- Consider what other alerts, teaching bubbles, or banners may be active on the page when crafting an onboarding experience.
+
+<Flex alignItems="center" flexDirection="column" my={3}>
+  <BorderBox width={8/12} border="1px solid" borderColor="red.5" mb="2" overflow="hidden">
+    <img src="https://user-images.githubusercontent.com/6744014/89232625-b0708500-d59c-11ea-8af1-249894f6143a.png"/>
+  </BorderBox>
+  <p><Text fontWeight="bold" color="red.5">Don't:</Text> forget to check coinciding features releases on the same page</p>
+</Flex>
+
+### Create efficient sequences
+
+Sequence tasks in a logical way to increase onboarding success. Clearly define when someone has started setup and successfully completed an onboarding flow. 
+
+Treat feature onboarding as a story or a journey with a Beginning, Middle, and End:
+
+#### Beginning: Draw attention to the feature
+
+- Invite the user to engage with an onboarding journey.
+- Use teaching bubbles, alert banners, the feature preview tray or annoucement modals to draw attention to the feature.
+
+<Flex my={3}>
+  <Box width="100%" mr="2">
+    <BorderBox border="1px solid" borderColor="green.5" mb="2" overflow="hidden">
+      <img src="https://user-images.githubusercontent.com/6744014/89225349-eb6bbc00-d58e-11ea-82b1-b84e8d35b1df.png" />
+    </BorderBox>
+    <Text fontWeight="bold" color="green.5">
+      Do:
+    </Text> use empty states or alert banners to call attention to a feature if users may discover the feature when starting a task.
+  </Box>
+  <Box width="100%">
+    <BorderBox border="1px solid" borderColor="green.5" mb="2" overflow="hidden">
+      <img src="https://user-images.githubusercontent.com/6744014/89225996-0c80dc80-d590-11ea-9e08-b86b2022b7eb.png" />
+    </BorderBox>
+    <Text fontWeight="bold" color="green.5">
+      Do:
+    </Text> use teaching bubbles or other easily dismissable elements if users will discover the feature when they are completing another task.
+  </Box>
+</Flex>
+
+#### Middle: Guide users through their task
+
+- Use form, checklist, or other components to help users complete their task.
+
+<Flex alignItems="center" flexDirection="column" my={3}>
+  <BorderBox width={9 / 12} mb={2} overflow="hidden">
+    <img src="https://user-images.githubusercontent.com/6744014/89254411-e8df8580-d5d3-11ea-969f-5080a8c6b0d4.png" />
+  </BorderBox>
+  <p>Example of a checklist.</p>
+</Flex>
+
+#### End: Celebrate the finish
+
+- Clearly define when someone has started setup and successfully completed an onboarding flow.
+- Use a toast, flash alert, or other messages to confirm the user has completed their task. Message should be short and brief.
+
+<Flex alignItems="center" flexDirection="column" my={3}>
+  <BorderBox width={9 / 12} mb="2" overflow="hidden">
+    <img src="https://user-images.githubusercontent.com/6744014/89254175-62c33f00-d5d3-11ea-8b95-7faaea7ae71b.png"/>
```

Comment:
This image has a funky border going on, might need a different aspect ratio?

Reply:
Yeah, couldn't figure it out after a while. Found adding a `display:block` fixes it but couldn't get it to compile in with gatsby....

Reply:
@ashygee are these following the guidelines for screenshots from the Figma template?